manifests/vanilla/csi-snapshot-validatingwebhook.yaml

```diff
@@ -0,0 +1,70 @@
+# Requires k8s 1.20+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: snapshot-validation-service
+  namespace: kube-system
```

Comment:
Shouldn't this be `vmware-system-csi` namespace

Reply:
This is an external validation webhook that is recommended to be deployed when using csi snapshot feature https://kubernetes-csi.github.io/docs/snapshot-validation-webhook.html , hence the kube-system namespace.

Reply:
Ok. I think `external-snapshotter` sidecar container will be eventually deployed as part of the CSI driver in `vmware-system-csi` namespace. However snapshot validation webhook will be in `kube-system`. I do not foresee any issues as such, but I thought we should deploy in `vmware-system-csi` for consistency.

Reply:
Both snapshot-controller and snapshot-validation-webhook are supposed to be deployed by a Kubernetes cluster distro.  So they should be deployed in the same namespace as kube-controller-manager. Unfortunately kubeadm has not picked this up as the development of snapshot feature is out of tree.

There's also an CSI external-snapshotter sidecar that will be deployed with the CSI driver and that should be in `vmware-system-csi`.

Reply:
Sounds good! Thanks